alloy.py

```diff
@@ -121,24 +121,14 @@ def checkout_LLVM(component, version_LLVM, target_dir, from_validation, verbose)
     GIT_REPO_BASE="https://github.com/llvm/llvm-project.git"
 
     # Identify the version
-    # An example of using branch (instead of final tag) is the following (for 9.0):
-    # git: "origin/release/9.x"
+    # An example of using branch (instead of final tag) is the following (for 10.0):
+    # git: "origin/release/10.x"
     if  version_LLVM == "trunk":
         GIT_TAG="main"
     elif  version_LLVM == "11_0":
         GIT_TAG="llvmorg-11.0.0"
     elif  version_LLVM == "10_0":
         GIT_TAG="llvmorg-10.0.1"
-    elif  version_LLVM == "9_0":
```

Comment:
Let's keep the ability to build LLVM - that might be useful sometimes. For example to bootstrap LLVM on old OS. I had to do that on macOS 10.10, where to build LLVM 10.0 you need to do that in several steps 6.0->8.0->10.0.

Reply:
Removed the alloy changes to build LLVM. Added change to disable appveyor run for 9.0 since it now fails. Waiting to see if it skips LLVM 9.0 now.

Reply:
Appveyor changes look good, please also Fix `.travis.yml`, it's straightforward enough to fix that without checking.

Reply:
Travis fixed too and tests passing.